vault/blog.2021.2021-09-27-markdown-on-mobile.md

```diff
@@ -1,66 +1,78 @@
+updated: 1634758861309
 created: 1632684719327
-date: '2021-09-27'
+date: '2021-10-20'
 image: "https://org-dendron-public-assets.s3.amazonaws.com/images/blog-mobile-editor-header.png"
-excerpt: "A stroll through mobile apps that can take notes in Markdown and make use of git"
+excerpt: "A stroll through mobile apps that can take Markdown-style notes"
 author: "Derek Ardolf"
 ---
 
 ![Photo of a phone ontop of a notebook](https://org-dendron-public-assets.s3.amazonaws.com/images/blog-mobile-editor-header.png)
 
-On the desktop, markdown-based notes can be taken in any plain-text editor of choice. Dendron sits ontop of VSCode/VSCodium. Though, what happens when you walk out the door? I know you look both ways before crossing the street with your open laptop, but it turns out that your phone can work with markdown, too!
+On the desktop, markdown-based notes can be taken in any plain-text editor of choice. If you're using [Dendron](https://www.dendron.so/), you can use VSCode/VSCodium as well as a suite of [powerful commands](https://wiki.dendron.so/notes/eea2b078-1acc-4071-a14e-18299fc28f47.html) to manage your notes. Though, what happens when you walk out the door? It turns out that your phone can work with markdown, too!
 
-Will these apps sway users to adopt them, and end up in news articles as the cause of distracted drivers? Please stop driving. Pull over, _right now_, and continue reading from that gas station parking lot.
-
-What we're looking at today:
+What we're looking at today, with a bias toward Android-compatible tooling due to the absence of Apple devices in my life:
 
+- [Obsidian](#obsidian)
 - [GitJournal](#gitjournal)
 - [Markor](#markor)
 - [iA Writer](#ia-writer)
 - [Other apps of interest](#other-apps-of-interest)
+  - [Markdown notes](#markdown-notes)
+  - [Git clients](#git-clients)
+- [Last thoughts](#final-thoughts)
+
+## Obsidian
+
+![Editor and markdown-preview screenshots, on Android, for Obsidian](https://org-dendron-public-assets.s3.amazonaws.com/images/blog-obsidian-screenshots.png)
+
+> _"Obsidian is a powerful knowledge base on top of a local folder of plain text Markdown files."_
+
+- **Cost:** Free (or [paid options](https://obsidian.md/pricing))
+- **OS:** Android, iOS / iPad / Mac, Windows
+- **Open Source:** No
+- **Built-in git integration:** No
+
+_[Obsidian](https://obsidian.md/mobile)_ is a PKM (Personal Knowledge Management) tool that provides a sleek UI to Markdown files. It had the best mobile experience for me, on Android, out of the apps I tested. Though it doesn't have git support built-in, other mobile apps can assist in providing that backend. I recommend [MGit (Android)](https://manichord.com/projects/mgit.html).
+
+Linking notes is supported in the [same fashion that Dendron uses](https://wiki.dendron.so/notes/9MZBqhrijEM4QpZRa5t08/) (ex. `[[my.other.note]]`), which is a nice plus.
+
+Paid options can assist note-takers with automatic syncing, publishing of notes, and other perks.
```

Comment:
Note that obsidian is free for personal use but requires a [paid license](link to their license) for business use.